logging/fb/fluent-bit_config.configmap_azmonitor.yaml

```diff
@@ -6,7 +6,7 @@ data:
     [FILTER]
        Name modify
        Match *
-       Set  fb_configMap_version 0.2.26
+       Set  fb_configMap_version 0.2.27noroot2a
```

Comment:
just curious as to what the “noroot2a” is? Maybe you explained it to me in the call but I can’t recall

Reply:
I increment the version number and add some text to help me track changes as I'm fixing things.  In our standard end-of-sprint work, we clean things up.  So, just prior to creating our next release, this version number will be change 0.2.27.   Basically, the stuff after the version number makes it clear there are changes in-flight and helps differentiate which changes have been made.  In this particular case, the "noroot" was to make it clear this was related to running FB as a non-root user and the 2a meant this was something like the 3rd variation I tried.